subscriptions/follow_up/automatic_alerts.rst

```diff
@@ -0,0 +1,91 @@
+=======================
+Set up automatic alerts
+=======================
+
+Now that your subscriptions are up and running, you want to stay up-to-date with your customers.
+Some automation would be appreciated since you would not want to go through the list of all your
+subscribers to check how things are going. This is what the *Automatic alerts* feature is for.
+
+For example, when customers subscribe to your magazine, you probably want to send them an email to
+welcome them and express your gratitude. Or, if the satisfaction rate of your customers drops below
+50%, you probably want to schedule a call with them to understand the reasons of their
+dissatisfaction.
+
+With **Odoo Subscriptions**, you can automatically send an email to your customers to welcome them,
+you can also create a "Call" task for one of your salespeople so that he/she can try to understand
+your customer's dissatisfaction and finally, why not automatically send satisfaction surveys by
+email after a month to your customers to evaluate your services? All of that is possible with our
+amazing application!
+
+Create a new automatic alert
+============================
+
+In this example, it is shown how to create a new automatic alert to automatically send satisfaction
+surveys to your customers by email after one month of subscription. Go to
+:menuselection:`Subscriptions --> Configuration --> Alerts`. From there, you can create a new
+alert.
+
+.. image:: media/create-a-new-automatic-alert.png
```

Comment:
This is also a screenshot on master. I would make sure the screenshots are on 13 :)

Reply:
Done 👍